package.json

```diff
@@ -5,7 +5,7 @@
     "dev:spotlight": "spotlight-sidecar",
     "dev:next": "next dev",
     "dev": "run-p dev:*",
-    "build": "next build",
+    "build": "dotenv -c production -- drizzle-kit migrate && next build",
```

Comment:
migrate should run only while merging to develop or release.
This should be handled in the CI.yml and release.yml